src/Tribe/Views/V2/Widgets/Widget_Abstract.php

```diff
@@ -289,4 +350,78 @@ protected function args_to_context( array $arguments, Context $context ) {
 
 		return $context_args;
 	}
+
+	/**
+	 * Handles gathering the data for admin fields.
+	 *
+	 * @since TBD
+	 *
+	 * @param int                 $field_name    The ID of the field.
```

Comment:
:no_entry_sign: Doc comment for parameter $field_name does not match actual variable name $arguments

<sub>hash: 11352066f681aeb73cb35643c25a5de3</sub>